src/scss/partials/_nav.scss

```diff
@@ -67,7 +67,7 @@ $nav-popover-padding          : spacing(x2);
             padding: 0;
             height: 90%; // if calc not supported, just make it 90% height
```

Comment:
Is it worth removing this now, think calc is fairly well supported.

Reply:
Yeh – I reckon we can now